src/components/common/TmDataError.vue

```diff
@@ -6,8 +6,8 @@
     <div slot="subtitle">
       Even though you're connected a full node, we can't display this data for
       you right now. Please try again later or
-      <a href="https://github.com/luniehq/lunie/issues">file a bug report</a>.
-      Apologies!
+      <a href="https://github.com/luniehq/lunie/issues">file a bug report</a> or
+      <a id="intercom-button" href="#">open a support chat</a>. Apologies!
```

Comment:
now we have 2 buttons with that ID. are there probles with that? a class didn't work?

Reply:
It should work as I see in https://developers.intercom.com/installing-intercom/docs/javascript-api-attributes-objects, so going to change it.

Reply:
Currently there is no way to test it because netlify build is not included in intercom domain whitelist, but it _should_ work :-)